.github/workflows/electron-linux.yml

```diff
@@ -0,0 +1,53 @@
+name: Electron Rebuild CI - Linux (x32 & x64)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        electron:
+          - 5
+          - 6
+          - 7
+          - 8
+        include:
+          # includes a new variable for each electron (essentially a switch statement) 
+          - electron: 5
+            nodeVersion: 70
+          - electron: 6
+            nodeVersion: 73
+          - electron: 7
+            nodeVersion: 75
+          - electron: 8
+            nodeVersion: 76
+        arch:
+          - ia32
+          - x64
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        os: ${{ matrix.os }}
+    - name: Node Ver (For debugging)
+      run: node -v
+    - name: Install packages
+      run: npm install --unsafe-perm
+    - if: matrix.arch == 'ia32'
+      name: Run extra command on ia32
+      run: sudo apt-get install gcc-multilib g++-multilib
+    - name: Run on ${{ matrix.os }} - Electron v${{ matrix.electron }} (Node ${{ matrix.nodeVersion }})
+      run: node "./node_modules/node-gyp/bin/node-gyp.js" rebuild --target=${{ matrix.electron }}.0.0 --arch=${{ matrix.arch }} --dist-url=https://electronjs.org/headers
```

Comment:
Not sure if the `electron-rebuild` didn't work out, but you could also try running it through NPX like
```
npx electron-rebuild -v=${{ matrix.electron }} -w node-sass
```
That skips the need for the global install

Reply:
Switched back and now can be back into the master!!! 🎉